Sources/StreamChatSwiftUI/ChatChannel/Composer/PhotoAttachmentPickerView.swift

```diff
@@ -52,7 +52,7 @@ public struct PhotoAttachmentCell: View {
     @Injected(\.images) private var images
     @Injected(\.fonts) private var fonts
     
-    @StateObject var assetLoader: PhotoAssetLoader
+    @ObservedObject var assetLoader: PhotoAssetLoader
```

Comment:
Changed to ObservedObject as it is sufficient since it is passed from PhotoAttachmentPickerView. Please let me know if there are any issues, and I will revert the changes.

Reply:
yes, it should be ObservedObject here, thx